src/App.jsx

```diff
@@ -14,33 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
```

Comment:
We can reduce the amount of useState's by creating something like  `const [sortType, setSortType] = useState('');` and global variables

```
const IS_SORTED_ALPHABETICALLY = 'alphabetically';
const IS_SORTED_BY_LENGTH = 'length';

```
The example provided in the video: 
https://mate.academy/learn/react/list-manipulation?section=video&videoId=5364&testTaskSlug=js_list_of_goods_js